site/public/mjs/sql-toolbox.js

```diff
@@ -24,7 +24,7 @@ async function runSqlQuery() {
         table.innerHTML = '';
 
         if (json.status !== 'success') {
-            error_mesage.innerText = json.message;
+            error_mesage.textContent = json.message;
```

Comment:
JSDom (the HTML mocking library that jest uses) does not support `innerText`. While we could mock in support for it, better to just move to using `textContent` which is supported, and as we go rewrite any usage of `innerText` to `textContent` as well.